test/fuzz/validated_input_generator.cc

```diff
@@ -0,0 +1,511 @@
+#include "validated_input_generator.h"
+
+#include "source/common/protobuf/visitor_helper.h"
+
+#include "xds/type/matcher/v3/cel.pb.h"
+#include "xds/type/matcher/v3/domain.pb.h"
+#include "xds/type/matcher/v3/http_inputs.pb.h"
+#include "xds/type/matcher/v3/ip.pb.h"
+#include "xds/type/matcher/v3/matcher.pb.h"
+#include "xds/type/matcher/v3/range.pb.h"
+#include "xds/type/matcher/v3/regex.pb.h"
+#include "xds/type/matcher/v3/string.pb.h"
+
+namespace Envoy {
+namespace ProtobufMessage {
+
+#define MYLOGLEV error
```

Comment:
It may be better to remove this macro and set it to info - less global clobber.
If needed, the test log can probably be searched for "pgv::".

Reply:
Fixed